cli/run/batchWarnings.ts

```diff
@@ -196,6 +196,9 @@ const deferredHandlers: {
 						}
 						stderr(bold(loc));
 					}
+					else if (warning.loc) {
+							info(yellow(`${rollup.relativeId(warning.loc.file)}:${warning.loc.line}:${warning.loc.column}`)+' error '+message);
+					}
```

Comment:
I'm. not happy about this being the only non-title being formatted in yellow, couldn't we stick with the formatting used in the previous case? Also why do you want to repeat the error message that has just been written in the title? Moreover, `file` is optional in `.loc`, and this only makes sense if `file` is actually present otherwise you would not know what the location means.

Also this repeats the way that locations are formatted from the previous section, but in a slightly inconsistent way with regard to spacing. Here would be one suggestion to replace everything starting from line 192 with less duplication:

```js
const id = warning.id || warning.loc?.file;
if (id) {
  let loc = relativeId(id);
  if (warning.loc) {
    loc += `: (${warning.loc.line}:${warning.loc.column})`;
  }
  stderr(bold(loc));
}
```

Reply:
Lukas, I changed the code as you suggested.
I won't be able to add tests.